groovy/classes/RegisterSolids.groovy

```diff
@@ -450,6 +450,24 @@ class RegisterSolids {
                 .flags(GENERATE_ROD)
                 .colorAverage()
                 .build()
+
```

Comment:
This section right here might need to change IDs in case of ID conflict and the upcoming material refactoring.